noir-projects/aztec-nr/aztec/src/context/avm.nr

```diff
@@ -45,6 +45,23 @@ impl AVMContext {
 
     #[oracle(timestamp)]
     pub fn timestamp(self) -> Field {}
+
     // #[oracle(contractCallDepth)]
     // pub fn contract_call_depth(self) -> Field {}
+
+    #[oracle(emitNoteHash)]
+    pub fn emit_note_hash(self, note_hash: Field) {}
```

Comment:
this kinna introduces two ways to do one thing, `push_new_note_hash` seems to be enough here?